### ISSUE TITLE
type for start and end should be Optional[datetime]

### ISSUE THREAD

**Issue by skewballfox** [ActivityWatch/aw-client#73]
Not a major issue, but currently the function signature for client.get_events reads
```python
(method) def get_events(
    bucket_id: str,
    limit: int = -1,
    start: datetime = None,
    end: datetime = None
) -> List[Event]
```
but despite the value of start and end being assigned to None by default, if you try to pass in parameters for start and end that are either None or a datetime, this causes Pylance to interpret that as an issue if type checking is set to basic or strict in vscode. 

This can be fixed by using the Optional type:
```python
from typing import Optional, List
...
def get_events(
    bucket_id: str,
    limit: int = -1,
    start: Optional[datetime] = None,
    end: [datetime] = None
) -> List[Event]
```
I could probably open a PR for this next week, though I'm currently under a time crunch.

**Comment by ErikBjare** [comment]
This is already the case on master:

https://github.com/ActivityWatch/aw-client/blob/82f30d7a59e09799a817ec09856ba01b747ee4b2/aw_client/client.py#L160-L166

Must be an old version?

**Comment by skewballfox** [comment]
currently using aw-client version `0.5.11` from Pypi using poetry for dependency management.

**Comment by ErikBjare** [comment]
@skewballfox I just released v0.5.12 for you :)
